src/main/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidator.java

```diff
@@ -1,12 +1,14 @@
 package edu.ksu.canvas.attendance.form;
 
+import org.apache.log4j.Logger;
 import org.springframework.stereotype.Component;
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 
 @Component
 public class CourseConfigurationValidator implements Validator {
 
+    private static final Logger LOG = Logger.getLogger(CourseConfigurationValidator.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)